Cross.toml

```diff
@@ -3,3 +3,5 @@ passthrough = [
 	"RUSTFLAGS",
 	"RUST_TEST_THREADS"
 ]
+[target.loongarch64-unknown-linux-gnu]
+image = "ghcr.io/loongarchlinux/archlinux:latest"
```

Comment:
Looks like Loongarch has been [supported by cross](https://github.com/cross-rs/cross?tab=readme-ov-file#supported-targets), is this custom image still needed?